### PR TITLE
DFR-3117 Refactor ConsentOrderApprovedController 

### PR DESCRIPTION
**Pull request by so99y**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DFR-3117

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by Gubbsy — APPROVED** [review]
LGTM!

**Review by al-hmcts — DISMISSED** [review]
LGTM!

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/BaseHandlerTestSetup.java [hunk 1]
```diff
@@ -95,4 +99,18 @@ protected FinremCallbackRequest buildFinremCallbackRequest(String testJson) {
             throw new RuntimeException(e);
         }
     }
+
+    protected void assertCanHandle(CallbackHandler handler, CallbackType expectedCallbackType, CaseType expectedCaseType,
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/BaseHandlerTestSetup.java:103]: This is a great idea. However, it doesn't feel like this is the right class for it. How about creating a new class e.g. Assertions. Going forward we could add to it as suggested here https://www.baeldung.com/assertj-custom-assertion

Comment (so99y) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/BaseHandlerTestSetup.java]: Updated.

**Review by jthmcts — APPROVED** [review]

**Pull request merged**